docs/packages/connect/methods/solanaGetAddress.md

```diff
@@ -0,0 +1,100 @@
+## Solana: get address
+
```

Comment:
Since we are going to merge this before official firmware lands master branch, I suggest adding something like 

```
> :warning: **BETA**: This api is unstable and may change!

```

to all readmes and console.warn into every newly added connect methods init member.
just to be sure that people don't start using it right away and we don't have to wrap our heads around backwards compatibility in the future.

Reply:
Good point. Added in https://github.com/trezor/trezor-suite/pull/9958/commits/5b886ffb3d989c4b945426d52e46f4a472ae0c56.